pkg/scheduler/cache/cache.go

```diff
@@ -514,6 +514,35 @@ func newDefaultQueue(vcClient vcclient.Interface, defaultQueue string) {
 	}
 }
 
+// newRootQueue init root queue
+func newRootQueue(vcClient vcclient.Interface) {
```

Comment:
The functions of `newRootQueue` and `newDefaultQueue`  seem to be the same, but the queue names are different. Can this be achieved using one function?